DataManagementSystem/private/FTS3Utilities.py

```diff
@@ -6,11 +6,20 @@
 import random
 import threading
 
+from DIRAC.ConfigurationSystem.Client.Helpers.Resources import getFTS3ServerName
 from DIRAC.DataManagementSystem.Client.DataManager import DataManager
 from DIRAC.FrameworkSystem.Client.Logger import gLogger
 from DIRAC.Core.Utilities.ReturnValues import S_OK, S_ERROR
 from DIRAC.Core.DISET.RPCClient import RPCClient
 
+
+def _getVoName(lfn):
+  try:
+    voName = lfn.split('/')[1]
```

Comment:
This utility shouldn't be in FTS3Utilities. There should be something equivalent in StorageBase